diff --git a/nvmath/device/common_mathdx.py b/nvmath/device/common_mathdx.py
index 7a553712..da607b8b 100644
--- a/nvmath/device/common_mathdx.py
+++ b/nvmath/device/common_mathdx.py
@@ -76,7 +76,7 @@ def check_cuda_home():
         ):
             CUDA_HOME = (os.path.join(conda_include, ".."),)
             # TODO: verify conda case.
-            CURAND_HOME = os.path.join(CUDA_HOME, "include")
+            CURAND_HOME = os.path.join(CUDA_HOME[0], "include")
             return

     # Try local
